tgui/packages/tgui/interfaces/CentcomPodLauncher.js

```diff
@@ -1,518 +1,1125 @@
+      <Button
+        content="Teleport"
+        icon="street-view"
+        onClick={() => act('teleportCentcom')} />
+      <Button
+        content={data.oldArea ? data.oldArea.substring(0, 17) : 'Go Back'}
+        disabled={!data.oldArea}
+        icon="undo-alt"
+        onClick={() => act('teleportBack')} />
+    </Fragment>
+  );
+};
+
+const TabDrop = (props, context) => {
+  const { act, data, config } = useBackend(context);
+  const { mapRef } = data;
+  return (
+    <Fragment>
+      <Button
+        content="Teleport"
+        icon="street-view"
+        onClick={() => act('teleportDropoff')} />
+      <Button
+        content={data.oldArea ? data.oldArea.substring(0, 17) : 'Go Back'}
+        disabled={!data.oldArea}
+        icon="undo-alt"
+        onClick={() => act('teleportBack')} />
+    </Fragment>
+  );
+};
+
+const PodStatusPage = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [compact, toggleCompact] = useLocalState(context, 'compact', 0);
+  return (
+    <Section fill width="100%">
+      <Flex>
+        {EFFECTS_ALL.map((list, i) => (
+          <Fragment key={i}>
+            <Flex.Item>
+              <Box color="label" mb={1}>
+                <b>{(compact === 1 && list.alt_label)
+                  ? list.alt_label
+                  : list.label}:
+                </b>
+              </Box>
+              <Box>
+                {list.list.map((effect, j) => (
+                  <Fragment key={j}>
+                    {effect.divider && (
+                      <span style={pod_grey}><b>|</b></span>
+                    )}
+                    {!effect.divider &&(
+                      <Button
+                        tooltip={effect.details
+                          ? (data.effectShrapnel
+                            ? effect.title
+                            +"\n"+data.shrapnelType
+                            +"\nMagnitude:"
+                            +data.shrapnelMagnitude
+                            : effect.title)
+                          : effect.title}
+                        tooltipPosition={list.tooltipPosition}
+                        tooltipOverrideLong
+                        icon={effect.icon}
+                        content={effect.content}
+                        selected={effect.soloSelected
+                          ? data[effect.soloSelected]
+                          : (data[effect.selected] === effect.choiceNumber)}
+                        onClick={() => data.payload !== 0
+                          ? act(effect.act, effect.payload)
+                          : act(effect.act)}
+                        style={{
+                          'vertical-align': 'middle',
+                          'margin-left': (j !== 0 ? '1px' : '0px'),
+                          'margin-right': (j !== list.list.length-1
+                            ? '1px'
+                            : '0px'
+                          ),
```

Comment:
I see what you did here, setting a bad precedent for `max-len/80`, although you could do something like this
```suggestion
                          'margin-left': (j !== 0 ? '1px' : '0px'),
                          'margin-right': (
                            j !== list.list.length-1 ? '1px' : '0px'
                          ),
```